nautobot/virtualization/forms.py

```diff
@@ -453,7 +453,7 @@ class VMInterfaceForm(NautobotModelForm, InterfaceCommonForm):
         queryset=VLAN.objects.all(),
         required=False,
         label="Untagged VLAN",
-        brief_mode=False,
+        depth=1,
```

Comment:
Let's make sure to document that everywhere `brief_mode=False` it must now be `depth=1` on `DynamicModelChoiceField` instances!

Reply:
See my other comments, I don't think this is in fact correct/necessary in the vast majority of cases.